app/src/main/java/com/concordia/mcga/activities/MainActivity.java

```diff
@@ -70,6 +70,7 @@
     private Handler handler;
     public static final int SPOT_REQUEST_CODE = 1;
     private DrawerLayout drawerLayout;
+    private Campus currentCampus = Campus.LOY;
```

Comment:
Campus location needs to be known in order show the correct schedule for "Shuttle Schedule" activity. It is also needed for drawing the shuttle outdoor path. This will be resolve with the CampusAwareness PR